content/articles/transferring-domain.markdown

```diff
@@ -25,7 +25,7 @@ To avoid the possibility of downtime, you should [point the name servers for the
 ## Caveats
 
 - A domain may only be transferred if it was registered more than 60 days ago and has not been transferred within the last 60 days.
-- It takes up to five (5) days for a transfer to complete and this process generally cannot be expedited, though some registrars may allow explicit  approval. Check out [our post on this topic](https://blog.dnsimple.com/2012/01/things-to-know-about-transferring-a-domain/) for more information.
+- It can take up to five (5) calendar days for a transfer to complete depending on the domains Top Level Domain (TLD). To learn more about how to avoid downtime during this period, check out [this article](/articles/before-transferring-domain) for more information.
```

Comment:
```suggestion
- It can take up to five (5) calendar days for a transfer to complete depending on the domain's Top Level Domain (TLD). To learn more about how to avoid downtime during this period, check out [this article](/articles/before-transferring-domain) for more information.
```